### PR TITLE
feat: add hello world example when running bootstrap

### PR DESCRIPTION
**Pull request by Scra3**
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

File: packages/cloud-toolkit/src/services/bootstrap.ts [hunk 1]
```diff
@@ -60,17 +86,13 @@ export default async function bootstrap(
...
-        envPath,
-        `FOREST_ENV_SECRET=${envSecret}
-TOKEN_PATH=${homedir()}`,
-      );
-    }
+    if (!fs.existsSync(envPath)) await generateDotEnv(envSecret);
+
+    const introspection = await httpForestServer.getIntrospection();
+
+    if (introspection.length !== 0) await generateHelloWorldExample(introspection);
```
Comment (Thenkei) [inline packages/cloud-toolkit/src/services/bootstrap.ts:93]: For information: This use case is not possible*

Comment (Scra3) [reply packages/cloud-toolkit/src/services/bootstrap.ts]: Yes probably, but I prefer to check :)

File: packages/cloud-toolkit/package.json [hunk 2]
```diff
@@ -34,7 +35,7 @@
     "cloud-toolkit": "dist/command.js"
   },
   "scripts": {
-    "build": "tsc",
+    "build": "tsc && cp -r src/templates ./dist",
     "build:watch": "tsc --watch",
```
Comment (realSpok) [inline packages/cloud-toolkit/package.json:39]: build:watch doesnt build the templates and causes this error:
✖ Bootstrap failed: ENOENT: no such file or directory, open '/Users/nicolas/Forest/agent-nodejs/packages/cloud-toolkit/dist/templates/.env.txt

File: packages/cloud-toolkit/src/templates/.env.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+FOREST_ENV_SECRET=FOREST_ENV_SECRET_TO_REPLACE
```
Comment (realSpok) [inline packages/cloud-toolkit/src/templates/.env.txt:1]:
```suggestion
FOREST_ENV_SECRET=<FOREST_ENV_SECRET_TO_REPLACE>
```
You could have `<>` surrounding replace patterns

**Review by realSpok — APPROVED** [review]

**Pull request merged**